packages/blue-sdk/src/vault/Vault.ts

```diff
@@ -293,6 +294,13 @@ export class AccrualVault extends Vault implements IAccrualVault {
     return MathLib.wMulDown(this.apy, MathLib.WAD - this.fee);
   }
 
+  /**
+   * Total interest paid by borrowers, increased with unrealized bad debt.
```

Comment:
```suggestion
   * Total interest paid by borrowers to this vault plus unrealized bad debt.
   * In the case of MetaMorphoV1.1 vaults, also contains realized bad debt.
```

Or we pass `totalAssets - lostAssets` to `accruedInterest`